terraform.tf

```diff
@@ -21,40 +21,47 @@ resource "github_repository" "terraform" {
   }
 }
 
-resource "github_branch_protection" "terraform-main" {
-  repository_id = github_repository.terraform.node_id
-
-  pattern                         = "main"
-  enforce_admins                  = true
-  allows_deletions                = false
-  require_signed_commits          = false
-  required_linear_history         = true
-  require_conversation_resolution = true
-  allows_force_pushes             = false
-  lock_branch                     = false
-
-  # Restrict merging PRs to admins
-
-  restrict_pushes {
-    blocks_creations = false
-    push_allowances = [
-      "nl-design-system/${github_team.kernteam-admin.name}",
-    ]
-  }
```

Comment:
Ik denk niet dat deze blokken nog nodig zijn. We hebben `github_repository_collaborators` waarin we mensen toegang geven toch, of mis ik iets?